app/src/main/res/drawable/ic_alert_black_24dp.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="24dp"
+    android:width="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path android:fillColor="#000" android:pathData="M13,14H11V10H13M13,18H11V16H13M1,21H23L12,2L1,21Z" />
```

Comment:
Doesn't really look good in AMOLED en Dark themes in my opinion.

Reply:
What doesn't look good, specifically? I noticed that the color black instead of white in dark and AMOLED themes, so I did fix that.

Reply:
It was just the icon!